src/Benchmarks/Middleware/JsonMiddleware.cs

```diff
@@ -42,21 +42,20 @@ public async Task Invoke(HttpContext httpContext)
                 httpContext.Response.ContentType = "application/json";
                 httpContext.Response.ContentLength = _bufferSize;
 
+#if !NETCOREAPP3_0
```

Comment:
Why?

Reply:
Looks like the IO feature is only needed without S.T.J